app/modules/sr/index.js

```diff
@@ -44,6 +45,14 @@ export default angular.module('xoWebApp.sr', [
 
     let {get} = xoApi
     $scope.$watch(() => xoApi.get($stateParams.id), function (SR) {
+      if (SR) {
+        forEach(SR.VDIs, vdi => {
+          vdi = xoApi.get(vdi)
+          if (vdi && isNumber(vdi.size)) {
```

Comment:
Yes, I used isNumber to avoid passing already human readable sizes and get 0 iB or something like that. I think the filter could handle that, yes.